### PR TITLE
[FIX]: Spacing mail text

### LINKED ISSUE #243: [FIX]: SSO user already exists
## Summary

Information given with the email and the default text is not correct, needs space between.

### Basic example

image

### Motivation

Making the info user friendly so its easier to understand

### PR DESCRIPTION
**Pull request by f-morgado**
Relates to #243 

## Screenshots (if visual changes)
image

## Proposed Changes

  - Spacing between text and email

This pull request closes #243

### CODE REVIEW

File: frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const SignUpOptionsForm: React.FC<SignUpOptionsFormProps> = ({
 			<Text size="md">
 				The email
```
Comment (dsousa12) [inline frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx:37]:
```suggestion
				The email&nbsp;
```

File: frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ const SignUpOptionsForm: React.FC<SignUpOptionsFormProps> = ({
...
 				<Text size="md" fontWeight="medium">
-					{emailName}
+					{''} {emailName} {''}
```
Comment (dsousa12) [inline frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx:39]:
```suggestion
					{emailName}
```

File: frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx [hunk 3]
```diff
@@ -36,7 +36,7 @@ const SignUpOptionsForm: React.FC<SignUpOptionsFormProps> = ({
...
 				</Text>
 				supports login with company SSO (Single Sign-on)
```
Comment (dsousa12) [inline frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx:41]:
```suggestion
				&nbsp;supports login with company SSO (Single Sign-on)
```

**Review by dsousa12 — CHANGES_REQUESTED** [review]

File: frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx (hunk 3 above)
Comment (f-morgado) [reply frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx:41]: Did it in a slightly different way, let me know if its ok :)

Comment (dsousa12) [reply frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx:41]: The way you have a space inside a stronger text. So I maintains my comment 😉

**Review by dsousa12 — CHANGES_REQUESTED** [review]

File: frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx [hunk 4]
```diff
@@ -34,11 +34,11 @@ const SignUpOptionsForm: React.FC<SignUpOptionsFormProps> = ({
 	return (
 		<Container direction="column">
 			<Text size="md">
-				The email
+				The email&nbsp;
 				<Text size="md" fontWeight="medium">
-					{emailName}
+					{emailName}&nbsp;
```
Comment (dsousa12) [inline frontend/src/components/auth/SignUp/SignUpOptionsForm.tsx:39]:
```suggestion
					{emailName}
```

**Review by dsousa12 — CHANGES_REQUESTED** [review]

**Review by dsousa12 — APPROVED** [review]

**Pull request merged**